frontend/src/services/validateIsvService.ts

```diff
@@ -18,6 +18,6 @@ export const postValidateIsv = (
       return res.data;
     })
     .catch((e) => {
-      throw new Error(e.response.data.message);
+      throw new Error(e.response.data?.message ?? e.message);
```

Comment:
question: if `data?.message` is an empty string we good with that?

Reply:
nope. I'll fix. Thanks!